src/logics/zeroex-v4/logic.swap-token.ts

```diff
@@ -81,7 +81,7 @@ export class SwapTokenLogic
         takerAddress: takerAddressInput,
       } = params;
       const slippagePercentage = slippage != null ? slippageToZeroEx(slippage) : undefined;
-      const takerAddress = takerAddressInput && (await this.calcAgent(takerAddressInput));
+      const takerAddress = takerAddressInput;
```

Comment:
Could we remove this line and also the above `takerAddressInput` as `takerAddress` can be directly used in the following `ZeroExQuote` request?